.github/workflows/ensure_runner_exists.yml

```diff
@@ -52,9 +52,7 @@ jobs:
         id: list-runners
         with:
           # Unfortunately there is no way to filter by label or status
-          route: GET /repos/{owner}/{repo}/actions/runners
-          owner: ${{ github.repository_owner }}
-          repo: ${{ needs.repo-name.outputs.name }}
+          route: GET /repos/${{ github.repository_owner }}/${{ needs.repo-name.outputs.name }}/actions/runners
```

Comment:
Trying to fix warning in job.

Reply:
From what I read the warning is a known issue.  I don't think you can fix it.